package/com.unity.formats.usd/Tests/Common/Data/UsdImportTests/UsdInstanceVertexTest.usda

```diff
@@ -0,0 +1,42 @@
+#usda 1.0
+(
+    doc = "Blender v3.3.1"
+    metersPerUnit = 1
+    upAxis = "Y"
+)
+
+def Xform "zAxisTest_Instanced"
```

Comment:
For the sake of reusability, I'd recommend calling this something generic rather than specifically about the test case. This is just a USD file containing a point instancer, and what we test with it is up to the test code. You could specify the orientation setting maybe, to distinguish it from other files.